contentcuration/contentcuration/frontend/shared/views/channel/ChannelDetailsModal.vue

```diff
@@ -64,23 +64,14 @@
     },
     data() {
       return {
+        dialog: true,
```

Comment:
Temporarily emulate a usual non-dialog page behavior (until we can update these modal components to regular pages)